#check that our program has an exception working correctly to handle an error

